packages/salesforcedx-vscode-apex/src/commands/apexActionController.ts

```diff
@@ -68,13 +70,12 @@ export class ApexActionController {
           // Step 4: Check if the file already exists
           const fullPath = await this.pathExists(openApiFileName);
           if (!fullPath) throw new Error(nls.localize('full_path_failed'));
-          // Step 5: Generate OpenAPI Document
-          progress.report({ message: nls.localize('generate_openapi_document') });
-          const openApiDocument = await this.generateOpenAPIDocument(eligibilityResult, context);
-
-          // Step 6: Write OpenAPI Document to File
-          progress.report({ message: nls.localize('write_openapi_document_to_file') });
-          await this.saveOasAsErsMetadata(openApiDocument, fullPath);
```

Comment:
can we leave this here and not make it part of the strategy? Because I think the storing is independent of the strategy

Reply:
It also looks like all of this will change in [W-17587198](https://gus.lightning.force.com/a07EE000027w9p8YAA), so let's leave it as is now.